junit-test/shared/src/test/scala/org/scalajs/junit/TestTest.scala

```diff
@@ -1,25 +0,0 @@
-package org.scalajs.junit
-
-import org.junit.Test
-import org.scalajs.junit.utils.{JUnitTest, SuccessFrameworkArgs}
-
-class TestTest {
-  @Test def onlyTest(): Unit = ()
-}
-
-class TestTestAssertions extends JUnitTest with SuccessFrameworkArgs {
-
-  override val expectedTotal: Int = 1
-
-  protected def expectedOutput(context: OutputContext): List[Output] = {
-    import context._
-    List(
-        testRunStartedOutput,
-        testStartedOutput("onlyTest"),
-        testFinishedOutput("onlyTest"),
-        successEvent,
-        testRunFinishedOutput,
-        done
-    )
-  }
```

Comment:
It is useless because:

- Testing a tests is something that is rarely done (when do you stop?)
- It doesn't even do this. Testing a test would mean to verify that a test fails if the code under test doesn't satisfy the conditions. This test definitely doesn't do that.
- Other tests test this as well.

I'm having the impression that this is just a relic from when the JUnit testing harness got bootstrapped.